Simplenote/AuthViewController+Swift.swift

```diff
@@ -17,9 +17,8 @@ extension AuthViewController {
         passwordField.placeholderString = Localization.passwordPlaceholder
         passwordField.delegate = self
 
-        // Forgot Password!
-        forgotPasswordButton.title = Localization.forgotAction.uppercased()
-        forgotPasswordButton.contentTintColor = .simplenoteBrandColor
+        // Secondary Action
+        secondaryActionButton.contentTintColor = .simplenoteBrandColor
```

Comment:
We're renaming `forgotPasswordButton` > `secondaryActionButton` since now it'll serve more than one purpose!